content/source/docs/cloud/api/ip-ranges.html.md

```diff
@@ -38,17 +40,22 @@ Name                             | Type   | Description
 
 -> **Note:** The IP Ranges API does not require authentication
 
+-> **Note:** This endpoint supports [If-Modified-Since][] HTTP request header
+
 `GET /meta/ip-ranges`
 
 Status  | Response                                        | Reason
 --------|-------------------------------------------------|----------
 [200][] | `application/json`                              | The request was successful
+[304][] | `application/json`                              | The request was successful, IP ranges were not modified since the specified date in `If-Modified-Since` header
```

Comment:
Which is the default case response code here? This feels like we get a 304 normally when there are no changes, and a 200 when there have been changes? Or am I misreading the intent? If I'm misreading it maybe we should make it clearer 😂

Reply:
Thinking harder about this - is a 304 only returned if you send the `If-Modified-Since` header with the request? Otherwise, 200?

Reply:
yes, the endpoint replies 304 with empty body, ONLY when the client includes the header `If-Modified-Since`, and the date is >= `Last-Modified`

otherwise, the response is 200.

Reply:
OK, LGTM then